src/api/routes/users/@me/index.ts

```diff
@@ -127,7 +127,7 @@ router.patch(
 		}
 
 		if (body.new_password) {
-			if (!body.password && !user.email) {
```

Comment:
?

Reply:
seems I had screwed up something with git, that's an old change, this pull request is dead anyhow, apparently this is the wrong spot for these assets, and currently there's no real correct spot for them either. No idea why that's there. I don't remember what exactly this does, but it looks to be changing users who exist not needing to enter their password while those who don't not needing to, to users that exist needing to enter their password and users that don't needing not to.